### PR TITLE
Updated status of rollout

### PR DESCRIPTION
**Pull request by anniecolonna**
Updated tagging by shift (to available but rolling out) and GCC status (to rolling out)

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-tags.md [hunk 1]
```diff
@@ -30,10 +30,10 @@ Tags in Microsoft Teams let users quickly and easily connect with a subset of pe
...
 
 > [!NOTE]
-> Tags are not yet supported in private channels. Tags are not yet available in US Government Community Cloud (GCC), GCC High, or Department of Defense (DoD) organizations. 
+> Tags are not yet supported in private channels. Tags are rolling out to US Government Community Cloud (GCC). Tags are not available in GCC High, or Department of Defense (DoD) organizations.
```
Comment (JohanFreelancer9) [inline Teams/manage-tags.md:36]:
```suggestion
> Tags are not yet supported in private channels. Tags are rolling out to US Government Community Cloud (GCC). Tags are not available in GCC High or Department of Defense (DoD) organizations. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copt review - minor edit.

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**